pointocracy.client/src/App.tsx

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useState } from 'react';
+import './App.css';
+
+interface Forecast {
```

Comment:
I feel like we can probably strip all this weather stuff out but it can be done in another PR if we don't want to do it right now

Reply:
I'll do that now.